site/conf.py

```diff
@@ -997,32 +997,37 @@
 CONTENT_FOOTER = """
 <div class="container">
   <div class="text-center">
-    <h1>{sponsors_heading}</h1>
-    <hr/>
-    <p><a class="btn btn-primary btn-lg active" href="{sponsor_link}">{sponsor_link_text}</a>
-    </p>
+    <h1 class="text-white">{sponsors_heading}</h1>
+    <a class="btn-primary btn-lg get-ticket-btn text-sm block my-4 font-bold max-w-sm mx-auto" href="{sponsor_link}">{sponsor_link_text}</a>
   </div>
-  <div class="row">
-    <div class="col-md text-center">
-        <img src="/logo.png" class="footer-logo" alt="PyCon Thailand" />
+  <div class="row mx-4 my-4">
+    <div class="row">
```

Comment:
Refer to this [SO thread](https://stackoverflow.com/a/34006377/4010864). This solves the issue stretching image issue on Chrome. We can also override `.row` which I'm not sure that will affect any other components of the site.

```suggestion
    <div class="row" style="align-items: flex-start;">
```

Reply:
OK, applied